src/server/api/lib/twilio.js

```diff
@@ -56,13 +56,16 @@ async function convertMessagePartsToMessage(messageParts) {
   const lastMessage = await getLastMessage({
     contactNumber
   });
+  if (!lastMessage) {
```

Comment:
Often SMS spam comes in.  If we failed to find a lastMessage, then this failed rather than returning and completing normally -- leaving a db connection open -- and possibly a transaction.